packages/server/graphql/resolveTypesForMutationPayloads.ts

```diff
@@ -0,0 +1,28 @@
+/*
+  By convention, a MutationPayload is a Union of ErrorPayload | MutationSuccess
+  e.g. type AddTaskPayload = ErrorPayload | AddTaskSuccess
+  When a mutation resolver returns a value, GraphQL needs to ascertain
+  whether the return value is an ErrorPayload or MutationSuccess
+  It does this by calling `resolveType` on the union.
+  This function finds all conventional MutationPayloads and
+  adds that `resolveType` if one is not already defined
+*/
+
+import {GraphQLSchema} from 'graphql'
+import {isUnionType} from 'graphql/type'
+
+const resolveTypesForMutationPayloads = (schema: GraphQLSchema) => {
+  Object.values(schema.getTypeMap()).forEach((type) => {
+    if (!isUnionType(type) || !type.name.endsWith('Payload')) return
```

Comment:
-1 we should also return if `type.resolveType` is already present. It is defined in `makeMutationPayload` and thus does not need to be redefined.

https://github.com/ParabolInc/parabol/blob/fcfb8a714245224f1339027427b5e928cecceb69/packages/server/graphql/types/makeMutationPayload.ts#L9